templates/plugin/dynflow.yml.erb

```diff
@@ -1,14 +1,6 @@
 ---
 :enabled: <%= @module_enabled %>
 :database: <%= scope.lookupvar('::foreman_proxy::plugin::dynflow::database_path') %>
-:core_url: <%= scope.lookupvar('::foreman_proxy::plugin::dynflow::core_url') %>
 
-# If true, external core will be used even if the core gem is available
-# If false, the feature will be disabled if the core gem is unavailable
-# If unset, the process will fallback to autodetection, using external core if the core gem is unavailable
-<% external_core = scope.lookupvar('::foreman_proxy::plugin::dynflow::external_core') -%>
-<% if [nil, :undefined, :undef].include?(external_core) -%>
-# :external_core: true
-<% else -%>
-:external_core: <%= external_core %>
-<% end -%>
+# Require a valid cert to access Dynflow console
+:console_auth: <%= scope.lookupvar('::foreman_proxy::plugin::dynflow::console_auth') %>
```

Comment:
Is this a regression that you're fixing?

Reply:
no, it was in core yml before.

or do you mean in a non-external-core scenario? heck, I don't know? ;)

I was following the lines of https://github.com/theforeman/smart_proxy_dynflow/commit/e205e19a00d8ef269953811a3c4e5140776915b1#diff-8a58542e75188035fcfefaf9049586f5665b71ae0dabe5f533a61d3719f00e78

Reply:
> or do you mean in a non-external-core scenario? heck, I don't know? ;)

Indeed. It looks like that wasn't implemented. However, it defaults to true in the plugin itself and is only used to disable authentication. Not really important to cherry pick then.